# Fundamental theory in genomic epidemiology

_One of the strengths of incorporating genomic data into epidemiological investigations is that it provides an additional, independent data stream by which to assess infectious disease dynamics. One of the challenges that comes hand-in-hand with that strength is that genomic epidemiology uses theory, analytical approaches, and jargon that surveillance epidemiologists may not be familiar with. In this Chapter we introduce the fundamental theory that underlies genomic epidemiological analysis and describe the terminology that genomic epidemiologists frequently use in describing and interpreting our analyses. This chapter, which summarizes the principles and mechanics of genomic epidemiology, should be pertinent to most readers of this handbook._


## The overlapping timescales of pathogen evolution and pathogen transmission.

Our ability to explore infectious disease dynamics using evolutionary analysis of pathogen genome sequences depends on a fundamental principle; pathogens evolve on roughly the same timescales as they circulate through a population of hosts. This principle means that the evolutionary trajectories of pathogens are shaped by the kinds of epidemiological and immunological forces that we, as public health practitioners, want to learn about. Pathogen genetic diversity becomes distributed in different ways depending on varying host movements, transmission dynamics, environments, and selective pressures, among other forces. Exploring those patterns can help us to understand to what extent these different factors shape epidemics.

In the following sections we will describe how mutations occur within a single infected individual and discuss how this leads to viral diversity observed at the population level, across multiple infected individuals in an outbreak.


### Viral diversity accumulates over the course of a single individual's infection.

In thinking about how pathogen diversity accumulates over an epidemic we begin with the processes occurring during  _one individual's infection_. In the example that follows we will consider an RNA virus. These pathogens are highly amenable to genomic epidemiology because they evolve rapidly and replicate to large population sizes. This enables us to visualize and discuss the dynamics of the evolutionary processes more readily.

To begin, imagine the index case of our theoretical viral outbreak. Upon infection, the virus enters into that person's cells, hijacking some of the host cell machinery to make the proteins that it will need to generate progeny virions. The virus must also copy it's RNA genome such that these genome copies can be packaged into the progeny virions. To do this, RNA viruses rely on a protein called the RNA-dependent RNA polymerase (RdRp). While our cells have polymerases that transcribe DNA to RNA, these won't work for making an RNA copy of an RNA genome. Thus almost all RNA viruses bring along their own RNA-dependent RNA polymerase for performing the task of replicating their genome.

Unlike our polymerases, the majority of RNA-dependent RNA polymerases lack proofreading capabilities. That means that during the process of genome replication, when the polymerase makes a mistake and incorporates the wrong nucleotide into the new genome it is transcribing, that base will stay there uncorrected, representing a change in the sequence of the "child" (the newly copied genome) compared to the "parent" (the template genome). These replication "typos" happen frequently; broadly speaking, one or two mutations like this occur _every single replication cycle_. With a random change occurring at one site in the genome pretty much every replication cycle, a genome length on the order of tens of thousands of sites, and millions or even billions of progeny virions being generated over the course of a single person's infection, pretty much every single change to a genome sequence that could occur will occur during a single person's infection. Thus, within a single infection, there is a large amount of pathogen genetic diversity, which we typically refer as **within-host diversity**.

### Stochasticity and selection influence variant frequency of within an infection.

The mutations that yield all this within-host viral diversity do not represent changes the virus is making _towards_ some trait. They are simply transcription errors, like typos that you might make while typing rapidly, and they will be distributed across the sites of the genome. Some of these mutations will have detrimental impacts, even lethal ones, that make a progeny virion less fit or even unviable. We refer to such mutations as **deleterious mutations**. Some mutations will have absolutely no effect on the fitness virion at all; we refer to these as **neutral mutations**. Some mutations could confer a fitness benefit to the progeny virion; these are **beneficial mutations**.

While the occurrence of mutations themselves is a random process governed primarily by the error rate of the RNA-dependent RNA polymerase, the impact that different mutations have on the ability of the progeny virion to infect cells and replicate will influence the frequency of those mutations within the diversity of an individual's infection. Mutations that are lethal or highly deleterious will be purged from the viral population quite quickly, as the progeny virions that carry those mutations fail to complete their replication cycle. Conversely, if a mutation is beneficial, perhaps it allows the virion to replicate more quickly, then the virion carrying that mutation will generate greater numbers of progeny, and the frequency or that mutation in the viral population will rise. Neutral mutations, which do not result in any functional changes to the virus, will rise or fall in frequency stochastically. Notably, deleterious to neutral to beneficial is a spectrum, and how significantly the mutation will change in frequency depends on how impactful the mutation is, as well as chance.

### When a transmission event occurs, the within-host viral diversity of the infector is sampled and transmitted to the recipient.

Over the course of a single individual's infection, mutations occur, generating slightly genetically-different viral populations within the infected person, and the frequency of these within-host variants rises, falls, or stays the same, as governed by chance and selective forces. But in genomic epidemiology our interest is typically in the process of transmission _between_ individuals, not in the process of a single individual's infection. So, what occurs at the time of transmission?

At the time of transmission, the within-host viral diversity of the infector is sampled and transmitted to the infectee. How many virions are "sampled" from the infector, and how many virions it actually takes to cause an infection in the recipient, varies from pathogen to pathogen. This concept is termed the **transmission bottleneck**. When transmission bottlenecks are narrow, just a small fraction of the viral diversity present within the infector is passed along to the recipient to initiate their infection. When transmission bottlenecks are wide, many virions are sampled from the infector and transmitted to the recipient.

While you do not need to know the width of the transmission bottleneck to conduct genomic epidemiological analysis, we bring up this concept because the width of transmission bottlenecks influence how genetic diversity gets passed along between individuals over the course of an outbreak. When the bottleneck is narrow, and just a small fraction of within-host diversity is passed along, random chance will play a significant role in which virions are transmitted and "found" the recipient individual's infection. In contrast, when the transmission bottleneck is wide, then the sample of the infector's within host diversity is more likely to mirror the extent and frequency of their within-host diversity, and the sample of viruses that founds the recipient's infection is likely to be similar to the infector's viral diversity at the time of transmission. Thus, the width of the transmission bottleneck influences how the viral diversity accumulating within a single infection gets passed along.

Because the within-host diversity of an infection is changing over the whole duration of that person's infection, transmission events occurring at different time points in the infector's infection will result in different samples of the viral diversity being transmitted. Similarly, because only a sample of the infector's viral diversity is transmitted on to a recipient, if a single infector were to infect many recipients at the same time, those recipient infections would be founded with slightly different samples of the infector's within host diversity as well.

### Consensus genomes provide a summary of the within-host diversity.

Despite the importance of within-host diversity to the overlapping timescales of pathogen evolution and transmission, in genomic epidemiology we most often look at a summary of within-host diversity, not the entirety of the diversity. This summary is the **consensus genome**. The consensus genome represents the _most frequently observed nucleotide at each site in the genome_ at the time of sample collection. At some sites in the genome there may be very little within-host diversity, and the vast majority of sequences support the same nucleotide. At other sites there might be higher levels of nucleotide diversity. In such cases this diversity can be summarized in the consensus genome with a [nucleotide ambiguity code](https://droog.gs.washington.edu/parc/images/iupac.html). These codes are letters that are not A, C, T, or G, but denote what nucleotide mixture was observed. For example, if at a site in the genome you have 40% of sequencing reads supporting an A and 60% of sequencing reads supporting a C, you might choose to summarize this diversity in your consensus genome by using the ambiguous site M, which means _A or C_. There are various decisions surrounding what thresholds of within-host diversity you would like to capture in a consensus genome sequence; a deep discussion of these tradeoffs and parameterizations of bioinformatic pipelines is beyond the scope of this introduction.

While the consensus genome sequence provides a summary of the within-host viral diversity at cross-section in time, it does not capture the full course of the within-host diversity. Moreover, the consensus genome does not capture all of the diversity present at the time of sampling. Many of the mutations that arise during the process of viral replication will remain at such low frequencies that a "real" mutation is not discernible from a mutation arising from PCR amplification or a sequencing error. As such, the consensus genome will not capture many of the mutations that occur over the course of an infection. This is one of the reasons why consensus genome sequences can be identical between closely linked infections, even though viruses are mutating with every replication cycle. This dynamic also means that the rate at which we _observe_ changes in the consensus genome sequence of different infections is different, that is slower, than the biologically-governed mutational rate of the pathogen. Because the rate at which we observe these changes is different from the actual fundamental mutation rate of a pathogen, we have varied terminology for describing these processes, that we describe in the next section of this chapter.


## Terminology for describing changes in genetic sequences.

Vocabulary can be tricky, and the meanings of different terms may vary between academic domains. Furthermore, you may hear multiple terms for describing the same phenomenon. Sometimes these terms are synonyms, and other times they may have distinct meanings. Here, we aim to provide some clarity surrounding terms used _in genomic epidemiology_ to discuss observed changes in microbial genetic sequences.

When dealing with microbial populations, the terms **mutation** and **SNP** (which stands for **s** ingle **n** ucleotide **p** olymorphism) are often used interchangeably. They refer to changes in the genetic sequence of the organism, at a single site in that organism's genome. You can observe a mutation or a SNP by comparing multiple aligned genetic sequences to each other. You may also hear mutations referred to as **alleles**, although this term can be confusing since it has a slightly different meaning when discussing the genetics of organisms that only carry one copy of their genetic material (such as viruses and bacteria) and the genetics of organisms that carry two or more copies of their genetic materials (such as humans).

The pattern of mutations that you observe across a sequence, summarized by the consensus genome sequence, is typically described as the **genotype**. Because viruses and bacteria are haploid, meaning they only carry a single copy of their genetic material, this single sequence defines their genotype. When you observe multiple samples with identical consensus genome sequences, we often describe these as multiple detections of the same genotype.

We use the term **substitution** to denote when a mutation has become completely dominant, and that all sequences in a particular population now carry that mutation. When this population-wide replacement happens, that mutation is said to be _fixed_ in the population. Knowing when a mutation has fixed, and therefore become a substitution, is challenging since it requires knowledge about the genetic diversity of the entire population. Therefore, while the term is used quite frequently, it may not always be used entirely correctly. You will likely encounter the term "substitution" most frequently when discussing the rate at which we expect to observe nucleotide changes accumulating; these rates are frequently referred to as **substitution rates**, although given the challenge of truly knowing when a mutation has become a substitution, they are most appropriately called **evolutionary rates**. We discuss these various rates, and how to measure them, in the next section of this chapter.

## Mutation rates, evolutionary rates, and molecular clocks
