<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 6 Case studies | An applied epidemiological handbook.</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 6 Case studies | An applied epidemiological handbook." />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="github-repo" content="alliblk/genepi-book" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 6 Case studies | An applied epidemiological handbook." />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Allison Black, Gytis Dudas, Patrick Ayscue" />


<meta name="date" content="2021-12-22" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="broad-use-cases-for-genomic-epidemiology.html"/>
<link rel="next" href="tools-and-methods.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> An introduction to this resource</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#what-is-this-handbook"><i class="fa fa-check"></i><b>1.1</b> What is this handbook?</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#what-this-handbook-is-not."><i class="fa fa-check"></i><b>1.2</b> What this handbook is not.</a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#who-is-this-handbook-written-for"><i class="fa fa-check"></i><b>1.3</b> Who is this handbook written for?</a></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#how-should-you-read-this-handbook"><i class="fa fa-check"></i><b>1.4</b> How should you read this handbook?</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><i class="fa fa-check"></i><b>2</b> The value of pathogen genomics in applied epidemiology</a><ul>
<li class="chapter" data-level="2.1" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-surveillance."><i class="fa fa-check"></i><b>2.1</b> The value of genomic epidemiology for surveillance.</a></li>
<li class="chapter" data-level="2.2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-outbreak-response."><i class="fa fa-check"></i><b>2.2</b> The value of genomic epidemiology for outbreak response.</a></li>
<li class="chapter" data-level="2.3" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-retrospective-data"><i class="fa fa-check"></i><b>2.3</b> The value of retrospective data</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="fundamental-theory-in-genomic-epidemiology.html"><a href="fundamental-theory-in-genomic-epidemiology.html"><i class="fa fa-check"></i><b>3</b> Fundamental theory in genomic epidemiology</a></li>
<li class="chapter" data-level="4" data-path="sample-selection.html"><a href="sample-selection.html"><i class="fa fa-check"></i><b>4</b> Sample selection</a><ul>
<li class="chapter" data-level="4.1" data-path="sample-selection.html"><a href="sample-selection.html#representative-sampling"><i class="fa fa-check"></i><b>4.1</b> Representative Sampling</a></li>
<li class="chapter" data-level="4.2" data-path="sample-selection.html"><a href="sample-selection.html#targeted-sampling"><i class="fa fa-check"></i><b>4.2</b> Targeted Sampling</a></li>
<li class="chapter" data-level="4.3" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data"><i class="fa fa-check"></i><b>4.3</b> Contextual data</a><ul>
<li class="chapter" data-level="4.3.1" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data-as-a-backdrop"><i class="fa fa-check"></i><b>4.3.1</b> Contextual data as a backdrop</a></li>
<li class="chapter" data-level="4.3.2" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data-as-controls"><i class="fa fa-check"></i><b>4.3.2</b> Contextual data as controls</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="broad-use-cases-for-genomic-epidemiology.html"><a href="broad-use-cases-for-genomic-epidemiology.html"><i class="fa fa-check"></i><b>5</b> Broad use cases for genomic epidemiology</a></li>
<li class="chapter" data-level="6" data-path="case-studies.html"><a href="case-studies.html"><i class="fa fa-check"></i><b>6</b> Case studies</a><ul>
<li class="chapter" data-level="6.1" data-path="case-studies.html"><a href="case-studies.html#are-cases-of-the-same-variant-of-concern-lineage-linked"><i class="fa fa-check"></i><b>6.1</b> Are cases of the same Variant of Concern lineage linked?</a></li>
<li class="chapter" data-level="6.2" data-path="case-studies.html"><a href="case-studies.html#evaluating-an-intake-screening-program-to-prevent-introduction-of-sars-cov-2-to-prisons."><i class="fa fa-check"></i><b>6.2</b> Evaluating an intake screening program to prevent introduction of SARS-CoV-2 to prisons.</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="tools-and-methods.html"><a href="tools-and-methods.html"><i class="fa fa-check"></i><b>7</b> Tools and methods</a></li>
<li class="chapter" data-level="8" data-path="further-concepts.html"><a href="further-concepts.html"><i class="fa fa-check"></i><b>8</b> Further concepts</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">An applied epidemiological handbook.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="case-studies" class="section level1">
<h1><span class="header-section-number">Chapter 6</span> Case studies</h1>
<p><em>In this Chapter, we provide case studies exemplifying how the broad thematic areas of genomic epidemiological analysis described in Chapter 5 can manifest in practice. These case studies illustrate step-by-step how different questions were investigated with a genomic epidemiological approach. Rather than providing the kinds of neat and cohesive narratives we frequently see in peer-reviewed literature, we have tried to show how hands-on investigations really proceed, including evaluating competing hypotheses and describing sources of uncertainty in our analyses. This chapter is pertinent to readers who will be directly involved in analyzing and interpreting genomic epidemiological studies, or who want to see examples of genomic epidemiology in practice.</em></p>
<div id="are-cases-of-the-same-variant-of-concern-lineage-linked" class="section level2">
<h2><span class="header-section-number">6.1</span> Are cases of the same Variant of Concern lineage linked?</h2>
<p>At the beginning of 2021, public health authorities within the United States were concerned about the transmission of the B.1.1.7 lineage (“Alpha” within the WHO nomenclature system), which appeared to be more transmissible than previous strains of SARS-CoV-2. Already the dominantly-circulating lineage within the United Kingdom, public health agencies within the United States were interested in whether B.1.1.7 had already arrived in the United States, and if so, how it was distributed and at what frequency. Genomic surveillance activities were established and intensified to investigate these questions. Beyond estimating frequency, some public health agencies also monitored for individual cases infected with “Variant of Concern” lineages, which they might prioritize for more aggressive contact tracing and control efforts to limit the establishment and growth of VOC lineages within the jurisdiction.</p>
<p>County A is a predominantly rural county, in which diagnostic testing and genomic surveillance are primarily handled by the local public health laboratory. The county started an in-house SARS-CoV-2 sequencing program at the beginning of 2021, and had regular sequencing and lineage assignment protocols in place during the spring of 2021, when they detected their first two cases of B.1.1.7 lineage-viruses among two residents of the county. The two cases were detected and sampled during the same week, a timeline that was consistent with one case infecting the other, resulting in an epidemiologically-linked pair. Beyond the timing of the two infections, epidemiological-linkage between the cases seemed possible since both cases were assigned the same lineage (B.1.1.7), and no other B.1.1.7 cases had been detected in County A up to this point. However, while Pango lineage assignments can provide a useful summary of different genetic lineages, most Pango lineages have genetic diversity within the lineage. Especially for lineages whose frequency grows significantly, such as B.1.1.7, there may be many different transmission chains of B.1.1.7 viruses circulating within different geographic areas. In such cases, phylogenetic analyses can provide higher resolution for refining relationships between cases.</p>
<p>In response to these detections, County A undertook a rigorous analysis of the whole genome sequences from these two cases. Firstly, since these were the first B.1.1.7 cases detected in the county, County A wanted to verify the accuracy of the lineage calls. To do so, they looked at the <code>.bam</code> files, which show the actual sequencing read data mapped to a reference genome. To confirm the quality of the B.1.1.7 lineage defining SNP calls, they looked at the total number of reads that covered the lineage-defining site, and looked for the particular SNP call in each of the reads (Figure 5.1). This process was repeated for all B.1.1.7 defining sites.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-a-bam"></span>
<img src="figures/06_county-a-bam.png" alt="A zoomed in picture of site 23063 in the nucleotide sequence, where the reference sequence has an A and the sequencing reads show that this infection has a T at this site. The A23063T mutation in the nucleotide sequence corresponds to an N501Y substitution in Spike protein, which is one of multiple lineage defining mutations for B.1.1.7. At this site there are 390 distinct, high quality sequencing reads that support this call. Furthermore, both forward and reverse reads detect this nucleotide, further demonstrating that this call is real." width="80%" />
<p class="caption">
Figure 6.1: A zoomed in picture of site 23063 in the nucleotide sequence, where the reference sequence has an A and the sequencing reads show that this infection has a T at this site. The A23063T mutation in the nucleotide sequence corresponds to an N501Y substitution in Spike protein, which is one of multiple lineage defining mutations for B.1.1.7. At this site there are 390 distinct, high quality sequencing reads that support this call. Furthermore, both forward and reverse reads detect this nucleotide, further demonstrating that this call is real.
</p>
</div>
<p>Next, County A took the two confirmed B.1.1.7 lineage viruses and imported the sequences into Nextclade in order to assess their quality. The sequences showed minimal numbers of Ns, no mixed sites, and a reasonable number of mutations as compared to the Wuhan-Hu-1 reference genome (Figure 5.2). This indicates that the sequences are high quality, and appropriate for analysis. Notably, the sequences do have frameshift mutations in them, which is why the Nextclade F metric is red (Figure 5.2). While sometimes real, these frameshifts are often a bioinformatic artifact from the consensus genome assembly pipeline. While the author of a genome sequence will typically have to fix or document a frameshift for public repositories to accept the sequence, one can still use them in phylogenetic pipelines that “strip” these frameshifts away.</p>
<p>Looking at the two sequences together in the Nextclade alignment viewer, we can see that both samples share some SNPs, as indicated by the colored bars (Figure 5.2). This makes sense since we know that both samples are B.1.1.7 lineage viruses. However, we can also see that there are differences between the two sequences as well (Figure 5.2). Each sample has numerous additional SNPs that are unique to the sample. This indicates that these samples are likely not directly related. The next paragraph will discuss that logic in more depth, and show how we can see that genetic divergence on the phylogenetic placement available in Nextclade.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-a-quality"></span>
<img src="figures/06_county-a-quality.png" alt="The two B.1.1.7 lineage viruses as visualized in Nextclade (sequence names are intentionally masked). Notably, all quality metrics show that the sequences are of high quality, except for the “F” metric, which indicates that a frameshift mutation has been detected. The sequence view shows that the two sequences share many SNPs, but also have some unique SNPs that are not found in the other sequence." width="80%" />
<p class="caption">
Figure 6.2: The two B.1.1.7 lineage viruses as visualized in Nextclade (sequence names are intentionally masked). Notably, all quality metrics show that the sequences are of high quality, except for the “F” metric, which indicates that a frameshift mutation has been detected. The sequence view shows that the two sequences share many SNPs, but also have some unique SNPs that are not found in the other sequence.
</p>
</div>
<p>In Nextclade we can take our two sequences and “graft” them onto a pre-inferred Nextstrain phylogeny in a process that is termed “phylogenetic placement”. The sequences are placed onto the tree according to the patterns of substitutions that the tree summarizes, and that your sequences have. The sequences of interest are placed onto the tree at the point where most of the SNPs in your genome sequence have also been observed in the tree. Then, any mutations that are unique to your sample, and not yet detected in the background tree, are shown as branch length leading from the tree to the sample of interest. In Figure 5.3A below, we can see that both sequences group in the Alpha lineage portion of the tree, consistent with their designation as B.1.1.7 lineage viruses. When we zoom in to the Alpha clade of the tree, we can see that despite both viruses receiving a B.1.1.7 assignment, they are quite diverged (Figure 5.3B). Indeed, these two B.1.1.7 sequences are separated by 13 SNPs (Figure 5.2 and Figure 5.3B). This is much more genetic divergence than we would expect to see if the sequences were epidemiologically-linked.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-a-placement"></span>
<img src="figures/06_county-a-phylo-placement.png" alt="A) Nextclade phylogenetic placement of the two B.1.1.7 sequences onto a background Nextstrain tree. Nextclade places both of these sequences on the tree within the Alpha clade, consistent with the Pango lineage assignments and our visual assessment of the sequencing reads. B) A zoomed in view of the two sequences within the Alpha lineage of the tree shows that they are placed onto different parts of the tree within the Alpha clade. You can count the number of nucleotides changes separating the two viruses by “walking the path” of the highlighted branches between the two samples, counting up the number of nucleotide changes observed on each segment of the path as you go. An important note is that the vertical axis has no meaning, thus you are only summing up the number of SNPs observed along the horizontal branches." width="80%" />
<p class="caption">
Figure 6.3: A) Nextclade phylogenetic placement of the two B.1.1.7 sequences onto a background Nextstrain tree. Nextclade places both of these sequences on the tree within the Alpha clade, consistent with the Pango lineage assignments and our visual assessment of the sequencing reads. B) A zoomed in view of the two sequences within the Alpha lineage of the tree shows that they are placed onto different parts of the tree within the Alpha clade. You can count the number of nucleotides changes separating the two viruses by “walking the path” of the highlighted branches between the two samples, counting up the number of nucleotide changes observed on each segment of the path as you go. An important note is that the vertical axis has no meaning, thus you are only summing up the number of SNPs observed along the horizontal branches.
</p>
</div>
<p>For an in-depth discussion of different thresholds of divergence, please see “Why sequencing is better at dismissing links than confirming them” in the Fundamental Theory chapter. However, we can also do a back-of-the envelope calculation to provide some context about what a 13 SNP difference means. Firstly, we should note that the average evolutionary rate of SARS-CoV-2 is roughly 24.5 substitutions across the entire genome per year (see Figure 5.4). This means that if we were to randomly sample two sequences that were sampled a year apart from each other, we would expect them on average to be 24.5 SNPs different from each other. If you were looking at the actual divergence between two sequences, the SNP difference counts would be whole numbers, but this rate is the average value if you were to repeat that sampling procedure many times over. If we take our two samples from County A that are separated by 13 SNPs and think about what this distance means in light of the evolutionary rate of the virus, we can say that roughly 6 months worth of transmission separates these two infections.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-a-clock"></span>
<img src="figures/06_county-a-clock.png" alt="The clock view from the global SARS-CoV-2 Nextstrain build maintained by the Nextstrain team (available at https://nextstrain.org/ncov/gisaid/global?l=clock). Each circle in this plot represents a sampled virus (a tip in the tree), and it’s position in the plot is determined by the date the virus was sampled and the number of SNPs that it has compared to the Wuhan-Hu-1 reference genome. The evolutionary rate estimate comes from the slope of a least-squares line fit through the data points." width="80%" />
<p class="caption">
Figure 6.4: The clock view from the global SARS-CoV-2 Nextstrain build maintained by the Nextstrain team (available at <a href="https://nextstrain.org/ncov/gisaid/global?l=clock" class="uri">https://nextstrain.org/ncov/gisaid/global?l=clock</a>). Each circle in this plot represents a sampled virus (a tip in the tree), and it’s position in the plot is determined by the date the virus was sampled and the number of SNPs that it has compared to the Wuhan-Hu-1 reference genome. The evolutionary rate estimate comes from the slope of a least-squares line fit through the data points.
</p>
</div>
<p>What does this mean from a public health standpoint? County A can be confident that these two cases are not linked - the genome sequences are too diverged for that to be likely. Thus, these two cases of B.1.1.7 more likely represent independent introductions of B.1.1.7 into the county. This means that these two cases do not represent a transmission event of B.1.1.7 within County A. Rather, these two cases were likely separately infected somewhere else outside the county, and then returned home where they were tested for COVID-19. At this point, case interviews could be helpful for discerning whether these cases recently traveled, or what exposure event may have led them to contract a lineage that had not previously been detected in County A. Furthermore, since these two cases do not represent sustained transmission within the county, aggressive contact tracing efforts might be warranted in order to prevent establishment of B.1.1.7 circulation in the county.</p>
</div>
<div id="evaluating-an-intake-screening-program-to-prevent-introduction-of-sars-cov-2-to-prisons." class="section level2">
<h2><span class="header-section-number">6.2</span> Evaluating an intake screening program to prevent introduction of SARS-CoV-2 to prisons.</h2>
<p>County C has a large jail housing a considerable number of pre-trial detainees. Given the challenge of controlling SARS-CoV-2 outbreaks in congregate settings, including prisons, public health officials in County C implemented a screening system for COVID-19. The jail had experienced previous outbreaks, and the intent of this screening program was to prevent new introductions of SARS-CoV-2 into the jail, which could seed further outbreaks. As part of the screening program, newly incarcerated individuals were tested for COVID-19 if they consented, and underwent a two week quarantine period prior to being moved into the standard housing units.
The jail recorded cases across multiple residential pods. Given the duration of time over which cases were observed, and the fact that multiple residential pods appeared to be affected, epidemiologists were interested in what processes were contributing to COVID-19 cases within the jail. Was the intake quarantine program ineffective, in which case newly-admitted persons were introducing SARS-CoV-2 into the jail? Or had a previous outbreak within the jail never fully ended?</p>
<p>Given the ability of pathogen genomic sequence data to differentiate between related and unrelated cases, investigators sequenced samples from cases who tested positive during their intake screening quarantine period, and from cases that were detected within the residential pods of the jail. Five sequences were collected from individuals who had tested positive during intake quarantine, and twenty-one sequences were collected from individuals who tested positive while residing in the jail. All of the sequences were assigned the same Pango lineage, thus the lineage information did not provide sufficient information to discern relationships between the samples. Therefore, investigators took these sequences and performed a phylogenetic analysis of jail sequences alongside contextual sequence data that had been collected from County C and other locales through representative surveillance sampling.</p>
<p>The genetic divergence tree in Figure 5.5 below shows the clade grouping 26 sequences sampled from incarcerated individuals (jail sequences are in yellow, contextual data are in grey). Looking at that clade we can see a few important findings. Firstly, some of the jail sequences are genetically diverged, and are more closely related to contextual sequences sampled from the broader community than they are to other sequences collected from incarcerated individuals. Secondly, there is one clade in which samples from the jail have identical consensus genome sequences and sequences that are very closely related to each other. Knowing that genetically-similar infections are more likely to be linked to each other, and that genetically-dissimilar infections are likely unrelated, we can surmise from this tree that some of these cases likely constitute an outbreak, while other cases among incarcerated individuals represent community-acquired infections that are not part of the outbreak.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-c-tree"></span>
<img src="figures/06_county-c-tree.png" alt="Phylogenetic tree of jail sequences and representatively-sampled contextual sequences. A) The top panel of this figure shows the clade within which all jail sequences group with other contextual sequences. Contextual sequences are shown in grey while jail sequences are coloured yellow. In this view you can see the broad distribution of some jail sequences across the entire clade, as well as one cluster of jail sequences which appear closely genetically-related. B) A zoomed-in view of the clade which clusters many jail sequences together. Many of the jail sequences within this cluster have identical genome sequences, and therefore appear stacked vertically along the root node of the outbreak clade. We see other jail sequences within this cluster that appear to have one or two additional nucleotide mutations." width="80%" />
<p class="caption">
Figure 6.5: Phylogenetic tree of jail sequences and representatively-sampled contextual sequences. A) The top panel of this figure shows the clade within which all jail sequences group with other contextual sequences. Contextual sequences are shown in grey while jail sequences are coloured yellow. In this view you can see the broad distribution of some jail sequences across the entire clade, as well as one cluster of jail sequences which appear closely genetically-related. B) A zoomed-in view of the clade which clusters many jail sequences together. Many of the jail sequences within this cluster have identical genome sequences, and therefore appear stacked vertically along the root node of the outbreak clade. We see other jail sequences within this cluster that appear to have one or two additional nucleotide mutations.
</p>
</div>
<p>Knowing that not all of these cases are genetically-related is the first step in evaluating the intake screening program. However, this information alone is not sufficient. What we really want to see is <em>which</em> of these 26 sequences come from cases detected during intake screening, and which cases come from incarcerated individuals living in the residential pods. To combine the genomic picture with this epidemiologic data, we can make use of the functionality in Nextstrain that allows us to overlay additional surveillance data onto a tree. In Figure 5.6 below, we’ve overlaid the phylogenetic tree with a tsv file that maps “case status” (either inmate or intake screening) to the “strain name” of the sequence in the Nextstrain tree. This allows us to then color the tree according to this new field. In the tree below, we can see that sequences from cases found during intake screening are colored in yellow, while cases detected within the jail are colored in blue.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-c-overlay"></span>
<img src="figures/06_county-c-metadata-overlay.png" alt="Metadata overlay onto Nextstrain phylogenetic tree differentiates sequences from incarcerated individuals (inmates) and from intake screening. On the left hand side of this figure is a screenshot of the metadata file formatted such that it can be dragged and dropped onto the tree. On the right, we see the same cluster as shown in Figure 5.5, but now jail sequences are coloured according to whether they were sampled from incarcerated individuals residing in pods (blue) or from individuals undergoing intake screening (yellow). Contextual sequences remain coloured in grey." width="80%" />
<p class="caption">
Figure 6.6: Metadata overlay onto Nextstrain phylogenetic tree differentiates sequences from incarcerated individuals (inmates) and from intake screening. On the left hand side of this figure is a screenshot of the metadata file formatted such that it can be dragged and dropped onto the tree. On the right, we see the same cluster as shown in Figure 5.5, but now jail sequences are coloured according to whether they were sampled from incarcerated individuals residing in pods (blue) or from individuals undergoing intake screening (yellow). Contextual sequences remain coloured in grey.
</p>
</div>
<p>This picture helps us understand the following. Firstly, cases detected through the intake screening program <strong>are not</strong> genetically-related to cases that were detected within the jail’s residential pods. This means that the observed transmission within the jail’s residential units <strong>was not</strong> initiated by any of the sequenced positive cases detected during intake screening. Secondly, intake cases are genetically similar to sequences sampled representatively from surrounding communities. This result supports a scenario in which newly admitted individuals were infected prior to their intake, with those infections discovered during the quarantine period. Thirdly, sequenced cases from intake screening are terminal tips in the tree. By that, we mean that there are no sequences in the tree that descend from infections detected during intake screening. This means that the quarantine period halted onward transmission of SARS-CoV-2.</p>
<p>In contrast, within the clade grouping all sequences sampled from incarcerated individuals living in the residential pods, we see that twelve cases are infected with an identical genotype that is basal to other related genotypes detected among the incarcerated individuals residing in pods (Figure 5.7). Additional samples from incarcerated individuals show additional substitutions accrued on top of the basal genotype. While some of this diversity could represent the transmission of within-host diversity during a super-spreading event, it is also consistent with substitutions accruing over the course of multiple transmission events. The latter scenario, that the accrued genetic diversity is the result of ongoing circulation within the jail, is more likely given that the cases among incarcerated individuals were detected over multiple months.</p>
<div class="figure" style="text-align: center"><span style="display:block;" id="fig:county-c-jail-cluster"></span>
<img src="figures/06_county-c-jail-outbreak.png" alt="Phylogenetic tree showing the outbreak within the residential pods of the jail. Twelve sequences sampled from incarcerated individuals in the residential pods show identical genome sequences, and are the basal genotype of this clade. There are five contextual sequences that also have this identical genotype. Further investigation of which cases these contextual sequences were sampled from may provide some information regarding how this outbreak was introduced to the jail. Three sequences have an additional unique nucleotide mutation, and one sequence has an additional two nucleotide mutations that are unique to that sample. At the top of this clade we see an additional clade that groups 4 jail samples that all share one additional nucleotide mutation on top of the basal sequence. This pattern of unique and shared diversity, in conjunction with the timeframe over which these samples were collected, suggests ongoing circulation of this clade within the jail over time." width="80%" />
<p class="caption">
Figure 6.7: Phylogenetic tree showing the outbreak within the residential pods of the jail. Twelve sequences sampled from incarcerated individuals in the residential pods show identical genome sequences, and are the basal genotype of this clade. There are five contextual sequences that also have this identical genotype. Further investigation of which cases these contextual sequences were sampled from may provide some information regarding how this outbreak was introduced to the jail. Three sequences have an additional unique nucleotide mutation, and one sequence has an additional two nucleotide mutations that are unique to that sample. At the top of this clade we see an additional clade that groups 4 jail samples that all share one additional nucleotide mutation on top of the basal sequence. This pattern of unique and shared diversity, in conjunction with the timeframe over which these samples were collected, suggests ongoing circulation of this clade within the jail over time.
</p>
</div>
<p>In terms of public health policy, these findings indicate that the intake screening program is serving its intended effect, namely, preventing new introductions of SARS-CoV-2 into the jail. While it is unclear how the outbreak was introduced into the prison, the most cases within the incarcerated population appear to be related to ongoing transmission within the jail, and thus interventions should focus on controlling transmission within the jail and its residential pods.</p>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="broad-use-cases-for-genomic-epidemiology.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="tools-and-methods.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/06-case-studies.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
