<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 4 Sample selection | An applied genomic epidemiological handbook.</title>
  <meta name="description" content="This is a short guide to the theory and practice of genomic epidemiology, oriented towards viral pathogens." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 4 Sample selection | An applied genomic epidemiological handbook." />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a short guide to the theory and practice of genomic epidemiology, oriented towards viral pathogens." />
  <meta name="github-repo" content="alliblk/genepi-book" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 4 Sample selection | An applied genomic epidemiological handbook." />
  
  <meta name="twitter:description" content="This is a short guide to the theory and practice of genomic epidemiology, oriented towards viral pathogens." />
  

<meta name="author" content="Allison Black, Gytis Dudas, Patrick Ayscue" />


<meta name="date" content="2021-12-23" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="fundamental-theory-in-genomic-epidemiology.html"/>
<link rel="next" href="broad-use-cases-for-genomic-epidemiology.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">An applied genomic epidemiology handbook.</a></li>

<li class="divider"></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> An introduction to this resource</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#what-is-this-handbook"><i class="fa fa-check"></i><b>1.1</b> What is this handbook?</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#what-this-handbook-is-not."><i class="fa fa-check"></i><b>1.2</b> What this handbook is not.</a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#who-is-this-handbook-written-for"><i class="fa fa-check"></i><b>1.3</b> Who is this handbook written for?</a></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#how-should-you-read-this-handbook"><i class="fa fa-check"></i><b>1.4</b> How should you read this handbook?</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><i class="fa fa-check"></i><b>2</b> The value of pathogen genomics in applied epidemiology</a><ul>
<li class="chapter" data-level="2.1" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-surveillance."><i class="fa fa-check"></i><b>2.1</b> The value of genomic epidemiology for surveillance.</a></li>
<li class="chapter" data-level="2.2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-outbreak-response."><i class="fa fa-check"></i><b>2.2</b> The value of genomic epidemiology for outbreak response.</a></li>
<li class="chapter" data-level="2.3" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-retrospective-data"><i class="fa fa-check"></i><b>2.3</b> The value of retrospective data</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="fundamental-theory-in-genomic-epidemiology.html"><a href="fundamental-theory-in-genomic-epidemiology.html"><i class="fa fa-check"></i><b>3</b> Fundamental theory in genomic epidemiology</a></li>
<li class="chapter" data-level="4" data-path="sample-selection.html"><a href="sample-selection.html"><i class="fa fa-check"></i><b>4</b> Sample selection</a><ul>
<li class="chapter" data-level="4.1" data-path="sample-selection.html"><a href="sample-selection.html#representative-sampling"><i class="fa fa-check"></i><b>4.1</b> Representative Sampling</a></li>
<li class="chapter" data-level="4.2" data-path="sample-selection.html"><a href="sample-selection.html#targeted-sampling"><i class="fa fa-check"></i><b>4.2</b> Targeted Sampling</a></li>
<li class="chapter" data-level="4.3" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data"><i class="fa fa-check"></i><b>4.3</b> Contextual data</a><ul>
<li class="chapter" data-level="4.3.1" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data-as-a-backdrop"><i class="fa fa-check"></i><b>4.3.1</b> Contextual data as a backdrop</a></li>
<li class="chapter" data-level="4.3.2" data-path="sample-selection.html"><a href="sample-selection.html#contextual-data-as-controls"><i class="fa fa-check"></i><b>4.3.2</b> Contextual data as controls</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="5" data-path="broad-use-cases-for-genomic-epidemiology.html"><a href="broad-use-cases-for-genomic-epidemiology.html"><i class="fa fa-check"></i><b>5</b> Broad use cases for genomic epidemiology</a></li>
<li class="chapter" data-level="6" data-path="case-studies.html"><a href="case-studies.html"><i class="fa fa-check"></i><b>6</b> Case studies</a><ul>
<li class="chapter" data-level="6.1" data-path="case-studies.html"><a href="case-studies.html#are-cases-of-the-same-variant-of-concern-lineage-linked"><i class="fa fa-check"></i><b>6.1</b> Are cases of the same Variant of Concern lineage linked?</a></li>
<li class="chapter" data-level="6.2" data-path="case-studies.html"><a href="case-studies.html#evaluating-an-intake-screening-program-to-prevent-introduction-of-sars-cov-2-to-prisons."><i class="fa fa-check"></i><b>6.2</b> Evaluating an intake screening program to prevent introduction of SARS-CoV-2 to prisons.</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="tools-and-methods.html"><a href="tools-and-methods.html"><i class="fa fa-check"></i><b>7</b> Tools and methods</a></li>
<li class="chapter" data-level="8" data-path="further-concepts.html"><a href="further-concepts.html"><i class="fa fa-check"></i><b>8</b> Further concepts</a><ul>
<li class="chapter" data-level="8.1" data-path="further-concepts.html"><a href="further-concepts.html#recombination"><i class="fa fa-check"></i><b>8.1</b> Recombination</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">An applied genomic epidemiological handbook.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="sample-selection" class="section level1">
<h1><span class="header-section-number">Chapter 4</span> Sample selection</h1>
<p><em>In this Chapter we describe two major approaches to sample selection for genomic epidemiological studies. We describe when you should use these different sampling strategies, focusing on which types of questions you may be interested in, and how the particular sampling strategy supports the goal of your investigation. This Chapter is pertinent for any readers who are actively seeking to implement genomic surveillance programs and integrate genomic epidemiology into their public health investigations. Readers who want to understand why different sampling strategies are useful will also benefit from reading these sections.</em></p>
<div id="representative-sampling" class="section level2">
<h2><span class="header-section-number">4.1</span> Representative Sampling</h2>
<p>In representative sampling, a set of specimens are selected for sequencing such that the pathogen genetic diversity in the sequenced sample set is representative of the pathogen genetic diversity circulating in the broader population. This means that the investigator should be able to detect the same suite of genotypes in their sample as circulate in the broader population, and that the frequencies with which they observe different genotypes in their sequenced dataset should reflect the frequencies with which those genotypes are found in the broader population.</p>
<p>To maintain these attributes and protect the validity of their dataset, the investigator must ensure that they do not accidentally enrich for certain genotypes by preferentially sequencing samples that have a particular diagnostic trait, a particular clinical presentation, or affect a particular demographic group.</p>
<p>Furthermore, to maintain a representative dataset, the investigator must also avoid systematically excluding certain genotypes that are circulating in the population. Genotype exclusion can occur when you lack sequences from a particular portion of your population that does not mix homogeneously with everyone else in your population. For example, if you have an under-served population that co-mingles, but has limited contact with other groups in your community, then a certain pathogen genotype could circulate primarily within that community. If that population lacks equitable access to testing, then you may not detect transmission within the community or sequence the circulating genotype(s). This situation would lead you to miss or underestimate the prevalence of those genotypes.</p>
<p>Generally, we recommend using representative sampling when exploring surveillance questions, such as:</p>
<ul>
<li><p>What is the frequency of this variant in my population?</p></li>
<li><p>How are frequencies of these different variants changing over time?</p></li>
<li><p>What is the spatial distribution of different pathogen genotypes?</p></li>
<li><p>When was this particular pathogen genotype introduced to this population, and how long has it circulated for?</p></li>
<li><p>How much pathogen diversity do we observe in our community, and how does this relate to pathogen diversity in other communities?</p></li>
</ul>
</div>
<div id="targeted-sampling" class="section level2">
<h2><span class="header-section-number">4.2</span> Targeted Sampling</h2>
<p>In targeted sampling we aim to sequence as many samples as possible from a particular population, outbreak, or transmission chain, in order to understand the specific genotypes and disease dynamics associated with that population or setting. Examples of when we use targeted sampling include:</p>
<ul>
<li><p>Investigating populations showing particular clinical presentations of a disease to see whether a specific pathogen genotype appears correlated with an altered disease presentation.</p></li>
<li><p>Exploring whether an outbreak in a localized setting, such as a workplace, school, or medical facility, is the result of transmission occurring within the setting. Alternatively, infections could be acquired in the broader community, and simply detected in the localized setting, for example due to increased screening in the facility.</p></li>
<li><p>Investigating a particular set of cases that report an epidemiologic link to determine whether they are indeed part of the same transmission chain.</p></li>
<li><p>Investigating individuals presenting with a second acute period of illness, to distinguish between a reinfection event and reactivation of latent disease.</p></li>
</ul>
</div>
<div id="contextual-data" class="section level2">
<h2><span class="header-section-number">4.3</span> Contextual data</h2>
<p>Targeted sampling focuses on deeply sampling particular settings, transmission chains, or infections that meet case definitions. Yet, to analyze those datasets appropriately, you need to include representatively-sampled genome sequences alongside your targeted samples. Within genomic epidemiology, we often refer to these other representative samples as <em>contextual data</em>. They provide a backdrop for what is happening more broadly outside of your densely sequenced target population, and can improve analytic inferences. Importantly, they also serve as controls, enabling you to see whether dynamics observed in your targeted population are unique to that population, or whether they are typical of the broader set of sequenced cases.</p>
<div id="contextual-data-as-a-backdrop" class="section level3">
<h3><span class="header-section-number">4.3.1</span> Contextual data as a backdrop</h3>
<p>Including contextual data in the analysis of transmission dynamics within a targeted setting enables the public health practitioner to see whether the pathogen genotypes associated with the outbreak circulated in the community before and/or after the outbreak. This contextual information can potentially clarify how an outbreak began, and when it has truly ended. Furthermore, including contextual data in a targeted analysis can help elucidate links between an outbreak in the targeted population and transmission in the broader community. That information can help the epidemiologist to see whether an outbreak amplified transmission and seeded it in the broader population. Additionally, if the bounds of an outbreak aren’t truly known, representative samples that appear related to targeted samples may indicate a connection to the outbreak that was not previously known. Finally, including sequences sampled over longer time periods will make estimates of the molecular clock more accurate, and often more precise. When the cases of interest within your targeted analysis occur over a short time window, as in the case of small, localized outbreaks, including contextual sequences sampled over longer periods of time will ensure that your time tree analyses remain accurate.</p>
</div>
<div id="contextual-data-as-controls" class="section level3">
<h3><span class="header-section-number">4.3.2</span> Contextual data as controls</h3>
<p>Imagine that you are interested in a cluster of illnesses that have a different disease presentation. You wonder whether a change in the pathogen itself might be responsible for the changed clinical manifestation. To investigate, you decide to sequence pathogen genomes from cases that meet a case definition for the new clinical presentation. In the absence of an association between pathogen genotype and disease presentation, you may find that all of these individuals with similar disease presentations are infected with distinct and diverse pathogen genotypes. However, what if the individuals who meet your case definition are infected with the same or similar genotypes? Does that mean that there is an association between the genotype and disease presentation?</p>
<p>Not necessarily. Only looking at sequences from individuals showing a specific clinical course would be like only looking at the people who got sick after eating at a “poisoned picnic”. If every sick case at the picnic ate the potato salad, then you might conclude that the potato salad is to blame. But what if <em>everyone</em> ate the potato salad, including your controls who did not get sick after the picnic? Then the potato salad is probably not the culprit. The validity of your study depends on the controls.</p>
<p>Similarly, representatively-sampled contextual sequences act as controls in genomic epidemiological analyses. They allow you to see which pathogen genotypes circulate in individuals <em>not</em> presenting with the altered clinical course. Much like the case where everyone at the picnic ate the potato salad, it is completely possible that a particular genotype is dominantly circulating in a community and causing infections with varied clinical presentations. If this is the case, then your targeted samples may all be infected with the same genotype, but your representatively-sampled contextual sequences will also show that same genotype.</p>
<p>Just like controls in traditional epidemiological studies, contextual sequences should be collected from individuals who would have been sequenced as part of your targeted sampling effort had they fit the criteria that is guiding your targeted sampling effort.</p>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="fundamental-theory-in-genomic-epidemiology.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="broad-use-cases-for-genomic-epidemiology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/04-sampling.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
