<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 2 The value of pathogen genomics in applied epidemiology | An applied epidemiological handbook.</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.24 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 2 The value of pathogen genomics in applied epidemiology | An applied epidemiological handbook." />
  <meta property="og:type" content="book" />
  
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="github-repo" content="alliblk/genepi-book" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 2 The value of pathogen genomics in applied epidemiology | An applied epidemiological handbook." />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Allison Black, Gytis Dudas, Patrick Ayscue" />


<meta name="date" content="2021-12-21" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="intro.html"/>
<link rel="next" href="fundamental-theory-in-genomic-epidemiology.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<script src="libs/accessible-code-block-0.0.1/empty-anchor.js"></script>
<link href="libs/anchor-sections-1.0.1/anchor-sections.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.0.1/anchor-sections.js"></script>




<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">A Minimal Book Example</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="intro.html"><a href="intro.html"><i class="fa fa-check"></i><b>1</b> An introduction to this resource</a><ul>
<li class="chapter" data-level="1.1" data-path="intro.html"><a href="intro.html#what-is-this-handbook"><i class="fa fa-check"></i><b>1.1</b> What is this handbook?</a></li>
<li class="chapter" data-level="1.2" data-path="intro.html"><a href="intro.html#what-this-handbook-is-not."><i class="fa fa-check"></i><b>1.2</b> What this handbook is not.</a></li>
<li class="chapter" data-level="1.3" data-path="intro.html"><a href="intro.html#who-is-this-handbook-written-for"><i class="fa fa-check"></i><b>1.3</b> Who is this handbook written for?</a></li>
<li class="chapter" data-level="1.4" data-path="intro.html"><a href="intro.html#how-should-you-read-this-handbook"><i class="fa fa-check"></i><b>1.4</b> How should you read this handbook?</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><i class="fa fa-check"></i><b>2</b> The value of pathogen genomics in applied epidemiology</a><ul>
<li class="chapter" data-level="2.1" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-surveillance."><i class="fa fa-check"></i><b>2.1</b> The value of genomic epidemiology for surveillance.</a></li>
<li class="chapter" data-level="2.2" data-path="the-value-of-pathogen-genomics-in-applied-epidemiology.html"><a href="the-value-of-pathogen-genomics-in-applied-epidemiology.html#the-value-of-genomic-epidemiology-for-outbreak-response."><i class="fa fa-check"></i><b>2.2</b> The value of genomic epidemiology for outbreak response.</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="fundamental-theory-in-genomic-epidemiology.html"><a href="fundamental-theory-in-genomic-epidemiology.html"><i class="fa fa-check"></i><b>3</b> Fundamental theory in genomic epidemiology</a></li>
<li class="chapter" data-level="4" data-path="broad-use-cases-for-genomic-epidemiology.html"><a href="broad-use-cases-for-genomic-epidemiology.html"><i class="fa fa-check"></i><b>4</b> Broad use cases for genomic epidemiology</a></li>
<li class="chapter" data-level="5" data-path="case-studies.html"><a href="case-studies.html"><i class="fa fa-check"></i><b>5</b> Case studies</a></li>
<li class="chapter" data-level="6" data-path="tools-and-methods.html"><a href="tools-and-methods.html"><i class="fa fa-check"></i><b>6</b> Tools and methods</a></li>
<li class="chapter" data-level="7" data-path="further-concepts.html"><a href="further-concepts.html"><i class="fa fa-check"></i><b>7</b> Further concepts</a></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/rstudio/bookdown" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">An applied epidemiological handbook.</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="the-value-of-pathogen-genomics-in-applied-epidemiology" class="section level1">
<h1><span class="header-section-number">Chapter 2</span> The value of pathogen genomics in applied epidemiology</h1>
<p><em>Have you ever wondered “What would be the benefit of having genomic surveillance capacity?” This chapter attempts to answer that question. In the following chapter we describe the utility of pathogen genomic data for public health surveillance and for outbreak response. We have distinguished between surveillance and outbreak response use cases for two reasons. Firstly, the turnaround times for which genomic data retain utility are different for surveillance versus outbreak response. Typically, outbreak response requires sequencing and genomic analysis to occur more rapidly. Secondly, the sampling frames that you would use for surveillance are different from what you would use for outbreak response. We will elaborate on sampling frames in Chapter 4 of this resource. Finally, we recognize that getting a genomic epidemiology program up and running can be slow and challenging. We therefore discuss how genomic data can remain useful even when they are not available in real time. This chapter should be pertinent to most readers, since it provides a broad rationale for why genomic surveillance and genomic epidemiology help support applied epidemiology activities.</em></p>
<div id="the-value-of-genomic-epidemiology-for-surveillance." class="section level2">
<h2><span class="header-section-number">2.1</span> The value of genomic epidemiology for surveillance.</h2>
<p>Genomic data provide additional resolution to determine relationships between cases. From a surveillance standpoint, this ability to delineate clusters of related cases more sensitively allows public health practitioners to see separate pathogen lineages circulating within a particular area of interest. This capacity allows the epidemiologist to see distinct chains of transmission even when they circulate in a population concurrently, a finding that can be challenging to see in case surveillance data alone.</p>
<p>What is the benefit of detecting these distinct transmission chains? Firstly, this additional layer of resolution enables the public health practitioner to detect the emergence or introduction of new lineages into their community or surveillance area, and distinguish introduction events from endemic transmission within the surveillance area. Developing a more precise understanding of how the processes of introduction and endemic transmission drive incidence within your area allows public health authorities to intervene in more precise and tailored ways. For example, if most cases within the surveillance area result from multiple introductions of distinct lineages that transmit only minimally within the surveillance area, then policies that seek to reduce the introduction of disease from other areas are likely to be more effective in reducing case counts. In contrast, if most cases are attributable to a single circulating transmission chain within the surveillance area, then public health policies focusing on reducing travel-acquired cases are likely to be less effective, and greater focus should be oriented towards public health interventions designed to interrupt transmission within the community.</p>
<div class="figure">
<img src="figures/02_epi_curve_with_seqs.png" alt="" />
<p class="caption">Figure 1: Epi curves with sequences</p>
</div>
<p>Beyond simply detecting distinct transmission chains, genomic surveillance data can enable the epidemiologist to see how different clusters contribute to the overall picture of transmission. For example, perhaps you detect four circulating genotypes within your community. Despite the existence of these four transmission chains, the majority of the cases in your community can be attributed to the circulation of just one of those transmission chains. In this case, the public health practitioner can focus attention on understanding what risk factors or demographic profiles appear most associated with the primary circulating transmission chain, with the hope that a more detailed understanding of that particular transmission chain will underscore more tailored and effective interventions.</p>
<p>Furthermore, evaluating the contribution of different pathogen genotypes to case counts over time can provide more precise situational awareness about how different variables shape the epidemiology of a pathogen. For example, we may be concerned about how travel and holiday behavior contribute to cross-border pathogen spread. Genomic surveillance data can enable us to investigate that question, monitoring the prevalence of particular genotypes in different countries and observing changes in genotype-specific incidence rates following major travel periods. Such analyses enabled epidemiologists to monitor the rise of the 20E lineage of SARS-CoV-2 viruses in Europe during the summer of 2020, attributing the rising dominance of the variant to holiday travel, rather than to increased viral fitness or transmissibility (<a href="https://www.nature.com/articles/s41586-021-03677-y" class="uri">https://www.nature.com/articles/s41586-021-03677-y</a>).</p>
<p>However, sometimes more fit lineages of a pathogen do emerge; here again genomic surveillance data can provide fast situational awareness. By monitoring genotype-specific incidence rates, we are well poised to detect an acceleration in transmission of a particular lineage. Enabled with high-resolution descriptive epidemiological information, we can follow up with analytical epidemiological studies to test hypotheses about why a particular strain is rising in frequency. This exact process was critical to detecting the emergence of B.1.1.7 (Alpha) in the United Kingdom, and attributing the rise in frequency to the actual increased transmissibility of B.1.1.7 lineage viruses. (<a href="https://www.nature.com/articles/s41586-021-03470-x" class="uri">https://www.nature.com/articles/s41586-021-03470-x</a>)(<a href="https://www.science.org/doi/10.1126/science.abg3055" class="uri">https://www.science.org/doi/10.1126/science.abg3055</a>)</p>
<p>Finally, a change in commonly-reported symptoms or illness severity associated with an increase in a particular genotype could indicate a change in the clinical presentation of illness, possibly associated with a genotypic change in the pathogen.</p>
<div class="figure">
<img src="figures/02_frequency_changes.png" alt="" />
<p class="caption">Figure 2: Frequency changes by strain</p>
</div>
</div>
<div id="the-value-of-genomic-epidemiology-for-outbreak-response." class="section level2">
<h2><span class="header-section-number">2.2</span> The value of genomic epidemiology for outbreak response.</h2>
<p>Pathogen genome sequences can help us detect or rule out linkages between cases. Similarly to how specific case definitions improve the sensitivity of traditional epidemiologic study designs, genomic data can support outbreak response by improving one’s ability to accurately classify which cases form an outbreak cluster. Below are some toy examples to clarify how this works, and to demonstrate how this information is useful in guiding public health practice.</p>
<p>In this first toy example, imagine that three skilled nursing facilities in your community are all experiencing outbreaks of a particular illness. As spaces of congregate living, typically housing more vulnerable people, such facilities may be more prone to higher incidence of disease. Thus you may ask yourself, are the outbreaks I see in these three facilities independent outbreaks, attributable simply to the higher risk setting? Or might these outbreaks actually be linked? Furthermore, are all of the cases detected within the facilities linked? Or do some cases form a cluster, while others have simply been detected due to enhanced screening?</p>
<p>Figure 3 below shows how genomic data can help elucidate some of these possibilities. From the surveillance data we can see that we have cases detected across all three SNFs. When we sequence the viruses from those infections, an interesting picture unfolds. Each SNF has a subset of cases with closely related genome-sequences (indicated here by colour). Each SNF also has cases whose genome sequences are distinct from each other and from the related infections found across all of the SNFs. This theoretical picture helps us understand a few things. Firstly, these are not three independent outbreaks, but rather one outbreak across three facilities. This may cause us to ask are residents transferred between facilities regularly? Do staff members work at multiple facilities? How can we limit inter-facility transmission? Secondly, not all of the cases that were detected across the SNFs are related to the outbreak. Rather, we have one primary cluster. Additionally, we have detected other cases of the disease that were likely acquired in the community, separately from the outbreak.</p>
<div class="figure">
<img src="figures/02_linking_cases.png" alt="" />
<p class="caption">Figure 3: Linking cases with genomic data</p>
</div>
<p>Similarly, genomic data can distinguish cases that are not linked, even if they are detected within the same setting or facility, or infect individuals with similar demographic profiles. Distinguishing between unrelated and related cases within a single setting can enable epidemiologists to more clearly see what precise factor is leading to transmission. Let us take another toy example, in this case a factory where you detected an elevated incidence of disease above baseline. The cases are distributed across the factory, among both managerial staff who primarily work in offices and individuals working in assembling products. All the cases have been detected around the same time period, meaning that temporal pattern supports a single outbreak across the entire facility. In that case, what factor is facilitating transmission? How will you act in order to interrupt transmission?</p>
<p>Figure 4 below shows how the addition of genomic data enables you to see that the cases among office-based managerial staff are unrelated to the cases occurring on the assembly line. While all cases occurred among employees of the factory, infections among managerial staff are genetically distinct from cases that occurred among assembly staff. Therefore, managerial infections appear to have been acquired externally from the workplace. In contrast, all cases among assembly staff are genetically related. Distinguishing the office staff cases from the case assembly staff cases allows you to see that any transmission occurring within the factory appears to be specific to the assembly area of the factory. This improves the sensitivity with which you can detect the factors or behaviors that are facilitating transmission, and hopefully making it easier to intervene.</p>
<div class="figure">
<img src="figures/02_distinguishing_cases.png" alt="" />
<p class="caption">Figure 4: Distinguishing cases with genomic data</p>
</div>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="intro.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="fundamental-theory-in-genomic-epidemiology.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/rstudio/bookdown-demo/edit/master/02-utility.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["bookdown-demo.pdf", "bookdown-demo.epub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
